lib/devise/token_authenticatable/model.rb

```diff
@@ -36,7 +36,13 @@ module Models
     #
     module TokenAuthenticatable
       extend ActiveSupport::Concern
-
+      
+      Devise::Models.config(self, :should_reset_authentication_token, :should_ensure_authentication_token)
```

Comment:
On step back... I think that it probably is not a good idea to put configuration options that are specific to this gem into devise directly - which is currently done with `Devise::Models.config(...)`.

So, you might consider putting the definition of the two configuration options into https://github.com/baschtl/devise-token_authenticatable/blob/master/lib/devise/token_authenticatable.rb.